Content.Shared/StatusEffect/StatusEffectsSystem.cs

```diff
@@ -64,7 +64,8 @@ private void OnHandleState(EntityUid uid, StatusEffectsComponent component, ref
                     }
 
                     var time = effect.Value.Cooldown.Item2 - effect.Value.Cooldown.Item1;
-                    TryAddStatusEffect(uid, effect.Key, time);
+                    //TODO: Not sure how to handle refresh here.
+                    TryAddStatusEffect(uid, effect.Key, time, true);
```

Comment:
It shouldn't even matter given the above `if` ensures that the effect is not already present. But either way, `refresh: true` is what you'd want just in case it doesn't work for whatever reason.